FlowCrypt/src/main/java/com/flowcrypt/email/api/email/model/IncomingMessageInfo.kt

```diff
@@ -84,6 +91,140 @@ data class IncomingMessageInfo constructor(
     return hasSomePart(MsgBlock.Type.PLAIN_TEXT)
   }
 
+  fun toInitializationData(
+    context: Context,
+    @MessageType messageType: Int,
+    accountEmail: String,
+    aliases: List<String>
+  ): InitializationData {
+    val toAddresses = arrayListOf<String>()
+    val ccAddresses = arrayListOf<String>()
+    val bccAddresses = arrayListOf<String>()
+    var body: String? = null
+
+    val folderType = FoldersManager.getFolderType(localFolder)
+    when (messageType) {
+      MessageType.REPLY -> {
+        when (folderType) {
+          FoldersManager.FolderType.SENT,
+          FoldersManager.FolderType.OUTBOX -> {
+            toAddresses.addAll(getTo().map { it.address.lowercase() })
+          }
+
+          else -> {
+            toAddresses.addAll(getReplyToWithoutOwnerAddress().ifEmpty { getTo() }
```

Comment:
getTo() is used several times in this `toInitializationData` function.
Better to declear variable like `val toRecipient = getTo()` and use `toRecipient` in those plaecs?